pkgs/top-level/all-packages.nix

```diff
@@ -17254,9 +17254,7 @@ in
 
   powerline-go = callPackage ../tools/misc/powerline-go { };
 
-  powerline-rs = callPackage ../tools/misc/powerline-rs {
-    inherit (darwin.apple_sdk.frameworks) Security;
-  };
+  powerline-rs = callPackage ../tools/misc/powerline-rs { };
```

Comment:
Darwin build fails with:
```
  = note: ld: framework not found Security
          clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
          
error: aborting due to previous error
```

I think we need to revert this change/

Reply:
Darn, I hoped crate2nix handled this for me. What crate is failing? powerline-rs or one of the dependencies?

Reply:
`powerline-rs`